pkg/project/stack/aws.config.yaml

```diff
@@ -46,6 +46,10 @@ region:
 #       # See lambda timeout values here:
 #       # https://docs.aws.amazon.com/lambda/latest/dg/configuration-function-common.html#configuration-timeout-console
 #       timeout: 15
+#       # set the amount of ephemeral-storage: of 512MB
+#       # For info on ephemeral-storage for AWS Lambda see:
+#       # https://docs.aws.amazon.com/lambda/latest/dg/configuration-ephemeral-storage.html
+#       ephemeral-storage: 512
```

Comment:
```suggestion
#       # set the amount of ephemeral-storage: of 512MB
#       # For info on ephemeral-storage for AWS Lambda see:
#       # https://docs.aws.amazon.com/lambda/latest/dg/configuration-ephemeral-storage.html
#       # Available Since nitric/aws@1.14.2
#       ephemeral-storage: 512
```